frontend_app_admin_workspace_user/build_app.sh

```diff
@@ -23,14 +23,27 @@ function logerror {
 
 
 dev=""
-if [ "$1" = "-d" ]; then
-    dev="-dev"
+only_utils=""
+
+while [[ "$#" -gt 0 ]]; do
```

Comment:
I would use a single bracket like this, for consistency, but this is fine as-is:
```suggestion
while [ "$#" -gt 0 ]; do
```

Reply:
what is the difference please?

will fix

Reply:
Differences are subtle. The double brackets are a bash-specific syntax while the single brackets are standard shell syntax. In this example both are identical.

If you are curious, this post lists https://stackoverflow.com/questions/13542832/difference-between-single-and-double-square-brackets-in-bash

My TLDR (though I could be wrong is this:

> Recommendation: always use []
>
> There are POSIX equivalents for every [[ ]] construct I've seen.
>
> If you use [[ ]] you:
>
>  -  lose portability
>  - force the reader to learn the intricacies of another bash extension. [ is just a regular command with a weird name, no special semantics are involved.

(the single bracket is way simpler, the portability issue probably don't matter that much for us since we already have some bashisms in our scripts and only test and run them with bash so we wouldn't even notice POSIX breakages)

Reply:
ok, thx

fixed